### DISCUSSION TITLE
AVX2 for FwdTrans/InvTrans

### DISCUSSION THREAD

**Discussion by ws-huangym** [fraunhoferhhi/vvenc#454] [Q&A]
Does have plan to use AVX2 for FwdTrans/InvTrans

**Comment by adamjw24** [comment]
Yes, please see https://github.com/fraunhoferhhi/vvenc/blob/master/source/Lib/CommonLib/x86/TrafoX86.h

**Reply by ws-huangym** [reply]
I'm sorry. I means that whether it will use AVX2 for DCT trans?

**Reply by adamjw24** [reply]
It does

**Reply by ws-huangym** [reply]
tks
